pkg/crc/validation/validation.go

```diff
@@ -58,18 +59,23 @@ func ValidateEnoughMemory(value int) error {
 }
 
 // ValidateBundlePath checks if the provided bundle path exist
-func ValidateBundlePath(bundle string) error {
-	if err := ValidatePath(bundle); err != nil {
+func ValidateBundlePath(bundlePath string) error {
+	if err := ValidatePath(bundlePath); err != nil {
 		if constants.BundleEmbedded() {
 			return fmt.Errorf("Run 'crc setup' to unpack the bundle to disk")
 		}
-		return fmt.Errorf("%s not found, please provide the path to a valid bundle using the -b option", bundle)
+		return fmt.Errorf("%s not found, please provide the path to a valid bundle using the -b option", bundlePath)
 	}
 
-	userProvidedBundle := filepath.Base(bundle)
+	userProvidedBundle := filepath.Base(bundlePath)
+	isUsingCustomBundle := strings.Contains(userProvidedBundle, constants.GetDefaultBundle())
 	if userProvidedBundle != constants.GetDefaultBundle() {
 		if !constants.IsRelease() {
-			logging.Warnf("Using unsupported bundle %s", userProvidedBundle)
+			logging.Warnf("Using unsupported bundlePath %s", userProvidedBundle)
+			return nil
+		}
```

Comment:
You can move  `isUsingCustomBundle := strings.Contains(userProvidedBundle, constants.GetDefaultBundle())` here as it's not used anywhere else. I'd name the variable `usingCustomBundle` or `isCustomBundle`, this is easier to read.